blues/smartdart.py

```diff
@@ -5,55 +5,161 @@
 import simtk.unit as unit
 import numpy as np
 from blues.ncmc import SimNCMC
-def zero_masses( system, firstres, lastres):
-    for index in range(firstres, lastres):
-        system.setParticleMass(index, 0*daltons)
-
-def beta(temperature):
-    kB = unit.BOLTZMANN_CONSTANT_kB * unit.AVOGADRO_CONSTANT_NA
-    kT = kB * temperature
-    beta = 1.0 / kT
-    return beta
-
-
-
-def forcegroupify(system):
-    forcegroups = {}
-    for i in range(system.getNumForces()):
-        force = system.getForce(i)
-        force.setForceGroup(i)
-        forcegroups[force] = i
-    return forcegroups
-
-def getEnergyDecomposition(context, forcegroups):
-    energies = {}
-    for f, i in forcegroups.items():
-        energies[f] = context.getState(getEnergy=True, groups=2**i).getPotentialEnergy()
-    return energies
+import mdtraj as md
+
+#TODO consider throwing a warning if particle choices
```

Comment:
@nathanmlim - you reviewed these changes?

@sgill2 - can you create an issue for discussion on this "TODO" item and explain it?